pkgs/applications/networking/sync/rsync/rrsync.nix

```diff
@@ -17,6 +17,7 @@ stdenv.mkDerivation {
 
   postPatch = ''
     substituteInPlace support/rrsync --replace /usr/bin/rsync ${rsync}/bin/rsync
+    substituteInPlace support/rrsync --replace /usr/bin/perl ${perl}/bin/perl
```

Comment:
perl is only used in the shebang, which are automatically patched in the Fixup phase. This is not needed?

```
# grep perl /nix/store/mlqrgi2rly20c1v2vnnf2glax4v334lq-rrsync-3.1.3/bin/rrsync 
#!/nix/store/ihgwxaqh6c80kh581vhw8lkgljd4mdnv-perl-5.30.1/bin/perl
```

Reply:
Well, if that works automatically in the fixupPhase, it would not have broken my backup processes…

Reply:
The proposed change is a no-op:
```
# nix-build -E '(import (builtins.fetchTarball "https://github.com/NixOS/nixpkgs/archive/7efeca1721994573434eda026eabd57435092c74.tar.gz") {}).rrsync' -o original
/nix/store/mlqrgi2rly20c1v2vnnf2glax4v334lq-rrsync-3.1.3
# nix-build -E '(import (builtins.fetchTarball "https://github.com/NixOS/nixpkgs/archive/c8527a8c2a48ac9b0ee5023179c1c0bf617bef4a.tar.gz") {}).rrsync' -o pr76749 
/nix/store/fwn97zk7li1yv2bynvwlaq23mxfcg713-rrsync-3.1.3
# diff -rs original/ pr76749/           
Files original/bin/rrsync and pr76749/bin/rrsync are identical
```